raw-ipa-lib/src/report.rs

```diff
@@ -0,0 +1,137 @@
+use crate::threshold::DecryptionKey as ThresholdDecryptionKey;
+use crate::threshold::{Ciphertext, RistrettoPoint};
+
+use std::collections::HashMap;
+use std::fmt;
+
+pub struct EncryptedMatchkeys {
+    match_keys: HashMap<String, Ciphertext>,
+}
+
+impl PartialEq for EncryptedMatchkeys {
+    fn eq(&self, other: &Self) -> bool {
+        any_matches(self.match_keys.values(), &other.match_keys.values())
+    }
+}
```

Comment:
Is there any case in which we would need to compare encrypted matchkeys?  Maybe it makes sense to inspect for test code, but does this need to be something we expose to users of this code?